pkg/daemonsetlock/daemonsetlock.go

```diff
@@ -34,11 +41,11 @@ func New(client *kubernetes.Clientset, nodeID, namespace, name, annotation strin
 }
 
 // Acquire attempts to annotate the kured daemonset with lock info from instantiated DaemonSetLock using client-go
-func (dsl *DaemonSetLock) Acquire(metadata interface{}, TTL time.Duration) (acquired bool, owner string, err error) {
```

Comment:
while I was here I removed the unused named returned variable (all return statements use local vars of the appropriate type)